### PR TITLE
Update to upstream version

### PR DESCRIPTION
**Pull request by cryptoseal86**

### CODE REVIEW

**Comment by cryptoseal86** [comment]
Checked with mainnet contracts and updated addresses/fixed some issues with renaming of contract events

File: src/config.json [hunk 1]
```diff
@@ -1,6 +1,14 @@
 {
-  "stakingEscrowAddress": "0xAB51fBDd4Faf6c691884B3A9b475E34E2092aE81",
-  "tokenAddress": "0x1F805c93A0AA3Ef7D17C43B7D2b316D101722F21",
-  "policyManagerAddress": "0xF1Fe01eb9dfDb1F43BEd97923E96176526525EF4",
-  "workLockAddress": "0x3ACBb734d2Cc0FDd366BC50f19aBf84C8E5C4822"
+  "mainnet": {
+    "stakingEscrowAddress": "0x33d28fcb3204b146ea8d5f059ebef8679f992deb",
```
Comment (cygnusv) [inline src/config.json]:
```suggestion
    "stakingEscrowAddress": "0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2",
```

File: src/config.json [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+    "tokenAddress": "0x4fe83213d56308330ec302a8bd641f1d0113a4cc",
+    "policyManagerAddress": "0xF1Fe01eb9dfDb1F43BEd97923E96176526525EF4",
```
Comment (cygnusv) [inline src/config.json]:
```suggestion
    "policyManagerAddress": "0x67E4A942c067Ff25cE7705B69C318cA2Dfa54D64",
```

File: src/config.json [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+    "workLockAddress": "0xe9778e69a961e64d3cdbb34cf6778281d34667c2"
```
Comment (cygnusv) [inline src/config.json]:
```suggestion
    "workLockAddress": "0xe9778E69a961e64d3cdBB34CF6778281d34667c2"
```

**Review by cygnusv — CHANGES_REQUESTED** [review]

File: src/config.json (hunk 1 above)
Comment (cygnusv) [reply src/config.json]: This needs to be the address of the proxy.

File: src/config.json (hunk 2 above)
Comment (cygnusv) [reply src/config.json]: Same.

File: src/config.json (hunk 3 above)
Comment (cygnusv) [reply src/config.json]: Checksum validation.

**Review by vepkenez — APPROVED** [review]

File: src/config.json [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
-  "stakingEscrowAddress": "0xAB51fBDd4Faf6c691884B3A9b475E34E2092aE81",
-  "tokenAddress": "0x1F805c93A0AA3Ef7D17C43B7D2b316D101722F21",
-  "policyManagerAddress": "0xF1Fe01eb9dfDb1F43BEd97923E96176526525EF4",
-  "workLockAddress": "0x3ACBb734d2Cc0FDd366BC50f19aBf84C8E5C4822"
+  "mainnet": {
+    "stakingEscrowAddress": "0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2",
+    "tokenAddress": "0x4fe83213d56308330ec302a8bd641f1d0113a4cc",
+    "policyManagerAddress": "0x67E4A942c067Ff25cE7705B69C318cA2Dfa54D64",
+    "workLockAddress": "0xe9778E69a961e64d3cdBB34CF6778281d34667c2"
+  },
```
Comment (KPrasch) [inline src/config.json:7]: :+1: Looks correct.

**Review by KPrasch — APPROVED** [review]

**Pull request merged**